cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting to connect to database...")
+        db_connect = None
+        attempt = 0
+
+        while not db_connect:
+            try:
+                db_connect = connections["default"]
+                db_connect.cursor()
+            except OperationalError:
+                attempt += 1
+                self.stdout.write(
+                    f"Database unavailable, waiting 1 second... "
+                    f"(Attempt {attempt})"
+                )
```

Comment:
You might want to add logging for these messages instead of just writing to stdout. This can be useful for debugging and monitoring purposes, especially in production environments.